code/modules/point/point.dm

```diff
@@ -114,7 +114,16 @@
 /// possibly delayed verb that finishes the pointing process starting in [/mob/verb/pointed()].
 /// either called immediately or in the tick after pointed() was called, as per the [DEFAULT_QUEUE_OR_CALL_VERB()] macro
 /mob/proc/_pointed(atom/pointing_at)
-	if(client && !(pointing_at in view(client.view, src)))
+	if(!client)
+		return FALSE
```

Comment:
Prevents basicmobs from pointing, you should make clientless mobs just skip directly to pointing, don't change this part